apps/wakunode2/app.nim

```diff
@@ -512,6 +512,46 @@ proc setupProtocols(node: WakuNode, conf: WakuNodeConf,
       pubsubTopics = conf.topicsDeprecated.split(" ")
     else:
       pubsubTopics = conf.topics
+
+    # Validate topics and update a node's ENR accordingly
+    let parsed_topics = pubsubTopics.mapIt(NsPubsubTopic.parse(it))
```

Comment:
minor, we use `camelCase`.
feel free to have a look
https://hackmd.io/@vac/main/%2F5oIVOuQcSM2WKrnDLUPcpg

Reply:
Rust habits... come back when I don't pay attention! good spot.

Reply:
Any Nim tools to automate this?

Reply:
> Any Nim tools to automate this? (formatting & style)

no that i know 😢